Anlab.Mvc/Services/IOrderService.cs

```diff
@@ -108,17 +108,26 @@ private List<TestItemModel> GetJoined(IList<TestItemPrices> prices, List<TestIte
         /// </summary>
         /// <param name="orderToUpdate"></param>
         /// <returns></returns>
-        public async Task OverwiteOrderWithTestsCompleted(Order orderToUpdate)
+        public async Task<string> OverwiteOrderWithTestsCompleted(Order orderToUpdate)

```

Comment:
@jSylvestre I don't like the idea that we just have to know the return value is a string containing errors.  How about creating a return object like OverwriteOrderResult and it can have an errors property which is a string array.   Then you can check if `result.Errors.length > 0` instead of having to do string logic.